orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/util/ArtifactUtils.java

```diff
@@ -230,38 +201,10 @@ public List<Artifact> getArtifactsForPipelineIdWithoutStageRef(
         .orElse(Collections.emptyList());
   }
 
-  private List<String> getExpectedArtifactIdsFromMap(Map<String, Object> trigger) {
-    List<String> expectedArtifactIds = (List<String>) trigger.get("expectedArtifactIds");
-    return (expectedArtifactIds != null) ? expectedArtifactIds : emptyList();
-  }
-
   public void resolveArtifacts(Map pipeline) {
     Map<String, Object> trigger = (Map<String, Object>) pipeline.get("trigger");
-    List<?> triggers = Optional.ofNullable((List<?>) pipeline.get("triggers")).orElse(emptyList());
-    Set<String> expectedArtifactIdsListConcat =
-        new HashSet<>(getExpectedArtifactIdsFromMap(trigger));
-
-    // Due to 8df68b79cf1 getBoundArtifactForStage now does resolution which
-    // can potentially return null artifact back, which will throw an exception
-    // for stages that expect a non-null value. Before this commit, when
-    // getBoundArtifactForStage was called, the method would just retrieve the
-    // bound artifact from the stage context, and return the appropriate
-    // artifact back. This change prevents tasks like CreateBakeManifestTask
-    // from working properly, if null is returned.
-    //
-    // reference: https://github.com/spinnaker/orca/pull/4397
-    triggers.stream()
-        .map(it -> (Map<String, Object>) it)
-        // This filter prevents multiple triggers from adding its
-        // expectedArtifactIds unless it is the expected trigger type that was
-        // triggered
-        //
-        // reference: https://github.com/spinnaker/orca/pull/4322
-        .filter(it -> trigger.getOrDefault("type", "").equals(it.get("type")))
-        .map(this::getExpectedArtifactIdsFromMap)
-        .forEach(expectedArtifactIdsListConcat::addAll);
-
-    final List<String> expectedArtifactIds = new ArrayList<>(expectedArtifactIdsListConcat);
+    List<?> expectedArtifactIds =
```

Comment:
This concept is not what we should be doing. This now makes triggers of any type have a required parameter, `expectedArtifactIds` when this is just an echo concept. I will have a follow up that makes this cleaner and simpler since this is due to an earlier PR